core/templates/services/extension-tag-assembler.service.ts

```diff
@@ -81,19 +81,26 @@ export class ExtensionTagAssemblerService {
   }
 
   formatCustomizationArgAttrs(
-      element: JQuery,
-      customizationArgs: InteractionCustomizationArgs
-  ): JQuery {
+      interactionHtml: string, customizationArgs: InteractionCustomizationArgs
+  ): string {
     const caBackendDict = (
       this._convertCustomizationArgsToBackendDict(customizationArgs)
     ) as Record<string, Record<string, Object>>;
     for (const caName in customizationArgs) {
       const caBackendDictValue = caBackendDict[caName].value;
-      element.attr(
-        this.camelCaseToHyphens.transform(caName) + '-with-value',
-        this.htmlEscaperService.objToEscapedJson(caBackendDictValue));
+      const angleBracketIndex = interactionHtml.indexOf('>');
```

Comment:
Is it possible to assemble this without string manipulation?

Reply:
Not sure, I wasn't able to find any simple Angular solution for this.

Reply:
Is it possible to keep the previous approach for now, until we figure a safe approach out?

Reply:
https://github.com/oppia/oppia/pull/14236/files#r757113965 I guess this should save us the trouble.

Reply:
Done, removed the changes for now.